### ISSUE TITLE
setTheme(..) doesn't work

### ISSUE THREAD

**Issue by KevinKrueger** [gismofx/toast_ui.blazor_calendar#50]
setTheme(..)  and
theme=….

does nothing with the Calendar.

**Comment by gismofx** [comment]
Yes, currently not functional. It was posing a challenge for me at the time. Looks like the theme changes in V2 might help.
